plugins/inputs/logfile/logfile.go

```diff
@@ -190,7 +190,7 @@ func (t *LogFile) FindLogSrc() []logs.LogSrc {
 			stateManager := state.NewFileRangeManager(state.ManagerConfig{
 				StateFileDir:    t.FileStateFolder,
 				Name:            filename,
-				MaxPersistItems: 1, // TODO: Base this on the number of threads
+				MaxPersistItems: 2, // TODO: Base this on the number of threads
```

Comment:
Going to have a merge conflict here with https://github.com/aws/amazon-cloudwatch-agent/pull/1724